smart-wallets/n-key-wallet/n-key-wallet.pact

```diff
@@ -0,0 +1,106 @@
+(define-keyset 'admin-keyset (read-keyset 'admin-keyset))
+
+(module n-key-wallet 'admin-keyset
```

Comment:
I think we might want to investigate making `n-key-wallet` a fungible itself (as opposed to _supporting_ other fungibles). This way the external UX is at least the same as `coin`.